package.json

```diff
@@ -6,14 +6,17 @@
     "author": "Mubaris",
     "license": "MIT",
     "devDependencies": {
+        "all-contributors-cli": "^4.3.0",
```

Comment:
We're using YARN in here, can you please add this dependency by doing `yarn add all-contributors-cli -D` instead of the old plain `npm install .... `